pygeo/parameterization/DVGeo.py

```diff
@@ -1237,6 +1239,30 @@ class in geo_utils. Using pointSelect discards everything in volList.
 
         return self.DV_listSectionLocal[dvName].nVal
 
+    def addGeoDVComposite(self, dvName, ptSetName=None, u=None, scale=None):
```

Comment:
Please add some comments or dosctrings briefly explaining that this is more of a preprocessing routine thats run on a single proc.